src/endpoints/transactions/transaction-action/transaction.action.service.ts

```diff
@@ -95,6 +95,19 @@ export class TransactionActionService {
         metadata.functionName = dataComponents[0];
         metadata.functionArgs = args;
       }
+
+      if (metadata.functionName === 'relayedTx' && metadata.functionArgs.length === 1) {
```

Comment:
add support also for `relayedTxV2`

Reply:
also how you will treat the case you inside the `innerTx` data field contains another `relayedTx@hexEncodedTx`

Reply:
because it's a recursive call, there can be as many inner layers as needed

Reply:
Yes but you have to stop at the first inner tx because after that transaction will fail and the protocol will not execute more than one inner tx